src/conf.c

```diff
@@ -391,6 +393,8 @@ int config__parse_args(struct mosquitto_db *db, struct mosquitto__config *config
 		}else if(!strcmp(argv[i], "-h") || !strcmp(argv[i], "--help")){
 			print_usage();
 			return MOSQ_ERR_INVAL;
+		}else if(!strcmp(argv[i], "-k")){
```

Comment:
Where possible I prefer to keep short command line arguments for common, frequently used options. This is an option that should not be used in production, so I'd like something that is more effort to type. Could you please change to `--tls-keylogging`.